src/pages/index.js

```diff
@@ -8,8 +8,14 @@ import Section from '../components/Section';
 import PopupWithImage from '../components/PopupWithImage';
 import PopupWithForm from '../components/PopupWithForm';
 import UserInfo from '../components/UserInfo';
+import Api from '../components/Api';
+import PopupWithConfirm from '../components/PopupWithConfirm';
 
-// Identify edit, add and close buttons as elements
+// Identify profile elements
+const userName = document.querySelector(selectors.profileName);
+const userProfession = document.querySelector(selectors.profileProfession);
+
```

Comment:
The description disappears after updating the profile

Reply:
Nice find! Thank you. It's fixed. 😊